### DISCUSSION TITLE
그래픽 카드를 사용 설정 관련

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#19] [제보하기]
여기다 질문할 내용이 아닐 수도 있기는 한데, 혹시나 Known Issue인가 해서 문의드려요.

회사 노트북에 그래픽 카드가 설치되어 있습니다.
그래서 메타몽이 그 그래픽카드(Geforce)를 사용하도록 강제로 설정을 해주었는데도 불구하고 사용을 안하네요. 

그래픽카드 사용 프로세스에 metamong.exe가 나타났다가 스페이스로 들어가면 사라지는데 혹시 동일 현상을 겪었거나, 식별된 조치 방법 같은게 있을까요?

옆자리 폴은 별다른 설정을 안해도 자동 설정 따라서 그래픽 카드를 잘 사용하는데 저만 그러네요.

**Comment by ACT-TJ** [comment]
자리에서 보여주셔서 처음 확인한 현상이고, 랩탑 사양에 따라 성능에 꽤 영향을 미칠 수 있는 부분으로 생각이 됩니다.
저도 따로 알아보고, 유니티에도 한번 문의를 해보면서 최대한 확인을 해 본 뒤 피드백 드릴 수 있도록 하겠습니다.

저희 테스트로는 확인이 힘들었을 부분인데 ㅠㅠㅠ 제보에 대해 다시 한 번 감사 드립니다.
